packages/flame/test/assets/images_test.dart

```diff
@@ -37,5 +37,11 @@ void main() {
         images.length,
       );
     });
+
+    test('throws when setting an invalid prefix', () {
+      final images = Images();
+
+      expect(() => images.prefix = 'adasd', throwsAssertionError);
```

Comment:
we really need a simple method to check the assertion message, otherwise checks like these can be really brittle

Reply:
We have, but I am stupid and forgot to use it, changed